schema/release-schema.json

```diff
@@ -2580,7 +2580,7 @@
           "openCodelist": true
         },
         "title": {
-          "description": "The title of the related process, where referencing an open contracting process, this field should match the tender/title field in the related process.",
+          "description": "The title of the related process, where referencing an open contracting (or planning) process, this field should match the tender/title field in the related process.",
```

Comment:
We can omit "open" here https://github.com/open-contracting/standard/issues/1409#issuecomment-1130229088

Reply:
Done in https://github.com/open-contracting/standard/commit/756b5202b2cea363b576b7618d186e281d01b554